### ISSUE TITLE
GFA alignment failed.

### ISSUE THREAD

**Issue by chenshixuan** [marbl/canu#606]
Hello, I'm running canu  with the following command:
```
canu \
-p Beta -d Beta \
 genomeSize=400m \
 useGrid=false \
 errorRate=0.025 \
 corOutCoverage=68 \
 corMhapSensitivity=normal \
 minReadLength=4000 \
 corMinEvidenceLength=2000 \
 -pacbio-raw  filter.fastq
```
However canu failed at  GFA alignment ，Here is part of canu.out:

```
 BEGIN ASSEMBLY
--
-- Unitigger finished successfully.
-- Found, in version 1, after unitig construction:
--   contigs:      1206 sequences, total length 350537604 bp (including 315 repeats of total length 15167831 bp).
--   bubbles:      0 sequences, total length 0 bp.
--   unassembled:  111993 sequences, total length 1314019630 bp.
--
-- Contig sizes based on genome size --
--            NG (bp)  LG (contigs)    sum (bp)
--         ----------  ------------  ----------
--     10     4312655             7    40134217
--     20     3282412            18    82070695
--     30     2055069            33   120063449
--     40     1271650            60   161021604
--     50      765230            99   200446282
--     60      402019           175   240285019
--     70      214015           311   280074788
--     80       85035           610   320030131
--
-- Finished stage 'unitigCheck', reset canuIteration.
--
-- Running jobs.  First attempt out of 2.
----------------------------------------
-- Starting concurrent execution on Wed Aug 30 17:32:21 2017 with 2829.423 GB free disk space (1 processes; 28 concurrently)

    cd unitigging/4-unitigger
    ./alignGFA.sh 1 > ./alignGFA.000001.out 2>&1

-- Finished on Wed Aug 30 17:32:21 2017 (lickety-split) with 2829.416 GB free disk space
----------------------------------------
--
-- Running jobs.  Second attempt out of 2.
----------------------------------------
-- Starting concurrent execution on Wed Aug 30 17:32:21 2017 with 2829.416 GB free disk space (1 processes; 28 concurrently)

    cd unitigging/4-unitigger
    ./alignGFA.sh 1 > ./alignGFA.000001.out 2>&1

-- Finished on Wed Aug 30 17:32:22 2017 (1 second) with 2829.416 GB free disk space
----------------------------------------
--
-- GFA alignment failed.
--
================================================================================
Don't panic, but a mostly harmless error occurred and Canu stopped.

Canu release v1.5 failed with:
  canu iteration count too high, stopping pipeline (most likely a problem in the grid-based computes)
```

**Comment by skoren** [comment]
There have been significant changes to this code between 1.5 and 1.6. If you are interested in the using the graph output, I'd suggest re-running the assembly part on your dataset with the 1.6 release (e.g. ```canu -assemble -pacbio-corrected Beta/asm.trimmedReads.fasta.gz```). 

You can also skip this step and get the fasta assembly from this run. alignGFA.sh looks for two files, ```4-unitigger/Beta.unitigs.aligned.gfa``` and ```4-unitigger/Beta.contigs.aligned.gfa```. If one of those doesn't exist, just symlink it to the same file without aligned (so ```4-unitigger/Beta.contigs.gfa``` for ```4-unitigger/Beta.contigs.aligned.gfa``` and re-launch Canu. It will will then finish the assembly without running alignGFA.sh.

**Comment by chenshixuan** [comment]
Thank you  for your detailed replay. I tried as your tips, the result have been outputed .I have a more question, what the difference between the  running alignGFA.sh step and without alignGFA.sh step?
Thank you very much!!!

**Comment by skoren** [comment]
The alignGFA.sh computes cigar strings for the GFA output files from Canu. Most tools don't yet use them but tells you exactly how two contigs or unitigs are aligned rather than the approximate overlap that is reported from the pre-alignGFA output.

**Comment by chenshixuan** [comment]
Thank you vrey much !

**Comment by skoren** [comment]
I'll assume this is resolved, if you re-run with 1.6 and alignGFA still fails, please open a new issue.
